.github/workflows/build.yml

```diff
@@ -43,14 +43,12 @@ jobs:
       - name: Fetch all tags and branches
         run: git fetch --prune --unshallow
 
-      - name: Install .NET SDK 2.1.x, 3.1.x, 5.0.x, and 6.0.x
+      - name: Install .NET SDK 6.0.x and 8.0.x
         uses: actions/setup-dotnet@v4
         with:
           dotnet-version: |
-            2.1.x
-            3.1.x
-            5.0.x
```

Comment:
I am as sure as I can be that this is going to break some of the tooling that we have in this build, specifically the usage of Wyam for documentation generation, so I don't think we can remove these just now.